WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -105,43 +108,52 @@ private extension ReviewsDashboardCard {
     }
 
     func reviewRow(for viewModel: ReviewViewModel, isLastItem: Bool) -> some View {
-        HStack(alignment: .top, spacing: 0) {
-            Image(systemName: "bubble.fill")
-                .foregroundStyle(viewModel.review.status == .hold ? Color.secondary : Color(.wooCommercePurple(.shade60)))
-                .padding(.horizontal, Layout.padding)
-                .padding(.vertical, Layout.cardPadding)
-
-
-            VStack(alignment: .leading) {
-                if let subject = viewModel.subject {
-                    Text(subject)
-                        .bodyStyle()
-                        .padding(.trailing, Layout.padding)
-                }
+        Button {
```

Comment:
Big diff in this chunk but it's essentially just:
1. Wrapping the previous View into Button so it's tappable
2. Small logic change on how bubble icon color is chosen (now based on read status instead of pending status)

Reply:
Some nit-picking: the top and bottom padding of the review item seem to be uneven:

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/1c8fa987-62d3-42eb-bb52-62dd7966e8d2" width=320 />

The reason is that you added a padding to the divider that is rendered only for non-last items.

I suggest using padding on the parent items rather than the child items. That would ensure all the padding is consistent and simplify the code, making it easier to parse. Something like this:

```Swift
Button {
    onViewReviewDetail(viewModel)
} label: {
    HStack(alignment: .top, spacing: Layout.padding) {
         Image(...) // No padding
         VStack(...) // No padding for any child items
    }
    .padding(.horizontal, Layout.padding)
    .padding(.vertical, Layout.cardPadding)
}
```

Reply:
The purpose for adding individual padding in the child items of the review content `VStack` is because the subject and review text needs padding on the right so the text doesn't stick to the right edge, but the Divider needs to stretch and stick to the right edge. I can't figure that out without specifying individual padding for each child item there. I suppose I can group the non-divider child items into their own `VStack` and set consistent padding for them there.

Let me think about it some more.

Reply:
Good points, thanks for the explanations!

> The purpose for adding individual padding in the child items of the review content VStack is because the subject and review text needs padding on the right so the text doesn't stick to the right edge, but the Divider needs to stretch and stick to the right edge:

I didn't realize this. In this case, you should probably group the review content and divider within a VStack and add the bottom padding to it. That way you will not lose the padding when the divider is not displayed. 

> I can't figure out those padding differences without specifying individual padding for each. I suppose I can group the non-divider child items into their own VStack and set consistent padding for them there.

Usually, we use the default padding 16 for horizontal padding. The text might be adjusted a bit due to line break, so it doesn't look strictly 16 from the design.

> It's also the same story with the Image. It wants its own vertical padding because without it, it sticks a bit too high compared to the text next to it, like this:

You're using `.top` alignment for `HStack`. Try `.firstTextBaseline` to see if it helps.